api/types/accesslist/member.go

```diff
@@ -106,3 +106,13 @@ func (a *AccessListMember) CheckAndSetDefaults() error {
 func (a *AccessListMember) GetMetadata() types.Metadata {
 	return legacy.FromHeaderMetadata(a.Metadata)
 }
+
+// IsEqual defines AccessListMember equality for use with
+// `services.CompareResources()` (and hence the services.Reconciler).
+//
+// For the purposes of reconciliation, we only care that the user and target
+// AccessList match.
+func (a *AccessListMember) IsEqual(other *AccessListMember) bool {
+	return a.Spec.Name == other.Spec.Name &&
+		a.Spec.AccessList == other.Spec.AccessList
```

Comment:
What's the difference between `a.Metadata.Name` and `a.Spec.Name` ?
Why are the other fields (like Reason) not relevant here?

Reply:
`a.Metadata.Name` is the name of this membership record
`a.Spec.Name` is the name of the user that this record binds to the AccessList

And for the purposes that we're using this (reconciling membership lists with Okta via the Okta API and/or SCIM) the only thing that really changes is the username or accesslist name.

(@mdwn - please chime in here if I'm getting this horribly wrong)